subprojects/packagefiles/sparrow/meson.build

```
@@ -0,0 +1,37 @@
+project(
+    'sparrow',
+    'cpp',
+    version: '0.0.2',
+    license: 'Apache 2.0',
+    default_options: [
+        'cpp_std=c++20',
+    ]
+)
+
+incdir = include_directories('include')
+
+compile_args = []
+deps = []
+if get_option('use_date_polyfill')
+  compile_args += ['-DSPARROW_USE_DATE_POLYFILL']
+  deps += dependency('date')
+  deps += dependency('tz')
+endif
+
+exclude_files = [
+    'array_data_concepts.hpp',
+    'dictionary_encoded_layout.hpp',
+    'reference_wrapper_utils.hpp',
+]
+install_subdir(
+    'include/sparrow',
+    subdir: 'sparrow',
+    exclude_directories: ['details'],
+    exclude_files: exclude_files,
+)
```

```suggestion
install_subdir(
    'include/sparrow',
    exclude_directories: ['details'],
    exclude_files: exclude_files,
    install_dir: get_option('includedir'),
    install_tag: 'devel',
    subdir: 'sparrow',
)
```
I think (please double check the files end up in the right location).

And add `meson_version: '>=0.60.0'` to the `project(…)` call.